petastorm/spark/spark_dataset_converter.py

```diff
@@ -139,19 +158,55 @@ def __len__(self):
         """
         return self.dataset_size
 
-    def make_tf_dataset(self):
+    def make_tf_dataset(
+            self,
+            batch_size=32,
+            prefetch=None,
+            num_epochs=None,
+            workers_count=4,
```

Comment:
change to None to reserve autotune in the future. mention current default in the docstring.